build/webpack/webpack.extension.web.config.js

```diff
@@ -106,6 +106,9 @@ const config = {
                 typeof process.env.IS_PRE_RELEASE_VERSION_OF_JUPYTER_EXTENSION === 'string'
                     ? process.env.IS_PRE_RELEASE_VERSION_OF_JUPYTER_EXTENSION
                     : 'true'
+            ),
+            VSC_JUPYTER_CI_TEST_GREP: JSON.stringify(
+                typeof process.env.VSC_JUPYTER_CI_TEST_GREP === 'string' ? process.env.VSC_JUPYTER_CI_TEST_GREP : ''
```

Comment:
Required to get `grep` working for Mocha in web tests.
No way to pass env variables into Mocha, but to inject them when building.